popd: add '-n' option; pushd: add '-n' option (#10435)
tldr-pages/tldr: pages/common/popd.md

`popd`

> - Remove the Nth directory (from the left of the list printed with `dirs`), couting starts at zero:

Reviewer: ```suggestion
- Remove the Nth directory (starting from zero to the left from the list printed with `dirs`):
```
I would suggest reverting this change as the old description seems more concise.

`popd +N`

> - Remove the Nth directory (from the right of the list printed with `dirs`), counting starts at zero:

Reviewer: ```suggestion
- Remove the Nth directory (starting from zero to the right from the list printed with `dirs`):
```

`popd -N`

> - Remove the 1st directory (from the left of the list printed with `dirs`), counting starts at zero:

Reviewer: ```suggestion
- Remove the 1st directory (starting from zero to the left from the list printed with `dirs`):
```
